lib/sequelize.js

```diff
@@ -199,6 +199,8 @@ module.exports = (function() {
     this.importCache = {};
 
     Sequelize.runHooks('afterInit', this);
+
+    Promise.prototype.sequelize = this;
```

Comment:
I know, that's why I mentioned it ;-).

I tried this apporach since it gives looser coupling to the CLS library, since we don't have to require cls anywhere in sequelize code. On the other hand it does however mean tighter coupling internally in sequelize, since the promise class suddenly needs to have knowlede about the sequelize options.

Perhaps we should just hardcode the namespace to be sequelize, and have a cls: true / false flag in the sequelize constructor